server/routes/teamEventAPI.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const db = require("../db/models");
+const auth = require("../middleware/auth");
+
+const router = express.Router();
+
+// 1 - Create a new team event type
+router.post("/api/team-event", auth, (req, res) => {
+  if (req.body.duration) {
```

Comment:
Just wondering why we are checking for duration attribute? Is duration the only attribute that's required to create a team event? Also should we make duration required in the schema then we don't have to check for it.  If not supplied creating new team event will fail, and it will be caught by the catch block.

Reply:
In eventAPI, it requires the duration attribute so I followed the same patterned here.  I agree there may be ways to optimize this. Once we start integrating with FE I will get a better idea. But for now I'm open to suggestions 👍🏼